examples/orientation_parser.html

```diff
@@ -0,0 +1,119 @@
+        <link rel="stylesheet" type="text/css" href="css/loading_screen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/loading_screen.js"></script>
+        <script src="js/ThreeLoader.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            
+            // Define initial camera position
+            // Coordinate can be found on https://www.geoportail.gouv.fr/carte
+            // setting is "coordonnée geographiques en degres decimaux"
+            var positionOnGlobe = { longitude: 2.33481381638492, latitude: 48.850602961052147, altitude: 100 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            // Instanciate iTowns GlobeView
+            var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+                handleCollision: false,
+            });
+           globeView.controls.minDistance = 30;
+
+            var menuGlobe = new GuiTools('menuDiv', globeView);
+
+            setupLoadingScreen(viewerDiv, globeView);
+
+            function addLayerCb(layer) {
+                return globeView.addLayer(layer);
+            }
+
+            // Add one imagery layer to the scene
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+
+            // Add elevation layer.
+            var promises = [];
+            promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addLayerCb));
+            
+            // Input data : a geoJson file, with point features, with orientation specific properties
+            var panoramics = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/immersive/exampleParis1/panoramicsMetaDataParis.geojson';
+            var layer = {
+                crsOut: globeView.referenceCrs,
+            };
+            // Input data are given in Lambert93 projection
+            itowns.proj4.defs('EPSG:2154', '+proj=lcc +lat_1=49 +lat_2=44 +lat_0=46.5 +lon_0=3 +x_0=700000 +y_0=6600000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+            // Add a promise to load oriented point datas
+            promises.push(
+                // first parse the JSON format
+                itowns.Fetcher.json(panoramics, {})
+                // then get geographic informations (coordinates) using GeoJson parser
+                .then(orientations => itowns.GeoJsonParser.parse(orientations, layer))
+                // then get orientation informations 
+                .then(collection => itowns.OrientationParser.parse(collection.features, layer))
```

Comment:
It's not really a parser anymore. To be more generic and reusable without the features, it would take a function that takes directly attitude and crs.

```js
.then(collection => {
    const attitude = collection.features.properties;
    const quaternion = itowns.quaternionFromAttitude.parse(attitude, globeView.referenceCrs));
    collection.features.quaternion = quaternion;
    return features;
}
```

Reply:
normally in itowns :
`parser` : transform file to `itowns.feature`
`convert` : `itowns.feature` to `THREE.Object`

Reply:
ok i've exposed re usable methods